<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VF Cash Node / Qt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/favicon</normaloff>:/new/prefix1/favicon</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QPushButton" name="update_node_version">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Node Version</string>
      </property>
     </widget>
     <widget class="QPushButton" name="master_resync">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Master Resync</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>331</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Configuration File / Editor</string>
      </property>
      <widget class="QTextEdit" name="config_edit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>311</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Node Statistics</string>
      </property>
      <widget class="QLabel" name="total_transactions">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Total Transactions:</string>
       </property>
      </widget>
      <widget class="QLabel" name="blockchain_size">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Blockchain Size:</string>
       </property>
      </widget>
      <widget class="QLabel" name="num_peers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Num Peers:</string>
       </property>
      </widget>
      <widget class="QLabel" name="difficulty">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Difficulty:</string>
       </property>
      </widget>
      <widget class="QLabel" name="rewards">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Rewards:</string>
       </property>
      </widget>
      <widget class="QLabel" name="circulating">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Circulating Supply:</string>
       </property>
      </widget>
      <widget class="QLabel" name="minted">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Minted Supply:</string>
       </property>
      </widget>
      <widget class="QLabel" name="node_version">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Node Version:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>541</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Node Rewards Address / Public Key</string>
      </property>
      <widget class="QLineEdit" name="rewards_address">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>521</width>
         <height>25</height>
        </rect>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="save_config">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Config</string>
      </property>
     </widget>
     <widget class="QPushButton" name="single_config">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>410</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Single-Threaded Config</string>
      </property>
     </widget>
     <widget class="QPushButton" name="multi_config">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>440</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Multi-Threaded Config</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minimal_config">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>470</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimal Hardware Config</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_config">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>350</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Config</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>390</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Preset Configurations:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="peers_sync">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Peers Sync</string>
      </property>
     </widget>
     <widget class="QPushButton" name="vote">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>180</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Vote</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="setdiff">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>0.031000000000000</double>
      </property>
      <property name="maximum">
       <double>0.240000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Minting Difficulty Vote:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_node">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start Node</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>20</width>
        <height>521</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="vgate">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>80</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>VGATE Exchange</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bihodl">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>110</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Bihodl Exchange</string>
      </property>
     </widget>
     <widget class="QPushButton" name="vfhome">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>VF Cash Homepage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="discord">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>190</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Discord</string>
      </property>
     </widget>
     <widget class="QPushButton" name="telegram">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Telegram</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Wallet</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>671</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Send Transaction</string>
      </property>
      <widget class="QLineEdit" name="topub">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>441</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>To Address:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>90</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="send_amount">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>110</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="maximum">
        <double>4294967.294999999925494</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="send_trans">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>110</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="frompriv">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>631</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>From Private Key:</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="frank">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>320</y>
        <width>261</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="newkey">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Generate New Address / Key Pair</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>671</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Quick Send (sends directly from your rewards address)</string>
      </property>
      <widget class="QLineEdit" name="qtopub">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>441</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>To Address:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>30</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="qamount">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>50</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="maximum">
        <double>4294967.294999999925494</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="qsend">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>50</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
      <widget class="QLabel" name="rbal">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>441</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Your Balance: 0 VFC</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Explore</string>
     </attribute>
     <widget class="QLineEdit" name="explore_address">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="explore_combo">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>0</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="explore_list">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>781</width>
        <height>541</height>
       </rect>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Miner</string>
     </attribute>
     <widget class="QPushButton" name="start_miner">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>540</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Start Mining</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="mine_threads">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>540</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
     </widget>
     <widget class="QPushButton" name="list_unclaimed">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>540</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>List Unclaimed</string>
      </property>
     </widget>
     <widget class="QPushButton" name="claim_unclaimed">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>540</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Claim Unclaimed</string>
      </property>
     </widget>
     <widget class="QListWidget" name="mined_list">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="open_minted">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>540</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <zorder>open_minted</zorder>
     <zorder>start_miner</zorder>
     <zorder>mine_threads</zorder>
     <zorder>list_unclaimed</zorder>
     <zorder>claim_unclaimed</zorder>
     <zorder>mined_list</zorder>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Peers</string>
     </attribute>
     <widget class="QLineEdit" name="agent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>540</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="peers_table">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>541</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
